lib/mri/neo4j/driver/bolt/pool.rb

```
@@ -46,9 +46,26 @@ def initialize(size:, options:, connect_factory:, clock: Internal::Clock.new)
+          # @created is bumped inside the create block (only on a successful
+          # factory call — TimedStack rolls back its own count if it raises).
+          @metrics_lock = Mutex.new
+          @created = 0
+          @leased = 0
+          @stack = ConnectionPool::TimedStack.new(size: size) do
+            @connect_factory.call(Thread.current[@auth_key]).tap { bump(:@created, +1) }
+          end
         end
 
+        # Connections currently checked out of this pool.
+        def in_use = @metrics_lock.synchronize { @leased }
```

RuboCop (1.88.0)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)